tool/tctl/common/proxy_command.go

```diff
@@ -0,0 +1,66 @@
+package common
+
+import (
+	"context"
+	"github.com/gravitational/kingpin"
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/gravitational/trace"
+	"io"
+	"os"
+)
+
+// ProxyCommand returns information about connected proxies
+type ProxyCommand struct {
+	config *service.Config
+	lsCmd  *kingpin.CmdClause
+
+	format string
+
+	stdout io.Writer
+}
+
+// Initialize creates the proxy command and subcommands
+func (p *ProxyCommand) Initialize(app *kingpin.Application, config *service.Config) {
+	p.config = config
+
+	auth := app.Command("proxy", "Operations with information for cluster proxies").Hidden()
+	p.lsCmd = auth.Command("ls", "List connected auth servers")
+	p.lsCmd.Flag("format", "Output format: 'yaml', 'json' or 'text'").Default(teleport.YAML).StringVar(&p.format)
+
+	if p.stdout == nil {
+		p.stdout = os.Stdout
+	}
+}
+
+// ListProxies prints currently connected proxies
+func (p *ProxyCommand) ListProxies(ctx context.Context, clusterAPI auth.ClientI) error {
+	proxies, err := clusterAPI.GetProxies()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	sc := &serverCollection{proxies, false}
```

Comment:
Should we make one flag for setting the `verbose` field? Like in the [node command](https://github.com/gravitational/teleport/blob/master/tool/tctl/common/node_command.go#L249).